### PR TITLE
Feat: Implement loss weights for the SequenceClassifier and SequencePairClassifier

### PR DESCRIPTION
**Pull request by dcfidalgo**
This PR adds a `loss_weights` parameter that allows one to give different weights to the labels in the CrossEntropyLoss function.

One has to specify the label name and its corresponding weight in the model config file. An example would be:
```yaml
architecture:
  type: sequence_classifier
  loss_weights:
    'my_first_label': 0.2
    'my_second_label': 0.8
  text_field_embedder:
...
```

### CODE REVIEW

**Comment by dcfidalgo** [comment]
@frascuchon [line 146](https://github.com/recognai/biome-text/pull/104/files#diff-f70434674b7b289db487b5d3bb1bde61R144-R146) will throw an `KeyError` if the label is not found in the vocabulary, and hence the initialization of the model will fail (that is before the training starts). Do you think it is still necessary to implement an explicit check before?

File: src/biome/text/models/sequence_classifier_base.py [hunk 1]
```diff
@@ -85,7 +89,7 @@ def __init__(
         # dropout for encoded vector
         self._dropout = Dropout(dropout) if dropout else None
         # loss function for training
-        self._loss = torch.nn.CrossEntropyLoss()
+        self._loss = torch.nn.CrossEntropyLoss(weight=self._get_loss_weights(loss_weights))
```
[the remark this answers is not in the document]
Comment (dcfidalgo) [reply src/biome/text/models/sequence_classifier_base.py]: :+1: see the popping in the `_get_loss_weights` method
